### DISCUSSION TITLE
Good software.

### DISCUSSION THREAD

**Discussion by Pinebelly** [boxgaming/qbjs#117] [General]
Thanks for putting this software together. It seems to work very well so far in the browser for basic programming. I am planning to use qbjs for algebra and pre-calculus mathematics and plotting. Should I update in the future of observations to contribute activity on general discussion or does the qbjs community prefer another forum?

**Comment by boxgaming** [comment]
Hi @Jawz-77, thanks for trying out QBJS and yes the General discussion category will be fine for any questions you have.

**Comment by GT9866** [comment]
please help me.what should I do, I can't run my program
, please tell me what should I do, it's crashing

**Reply by boxgaming** [reply]
Perhaps you could share a little more information.  What error are you getting?  Do you have a code sample that is erroring?

**Reply by GT9866** [reply]
this did not happen before, the site did not hang, did not glitch, and did not crash, even print "hello, world" glitches and hangs there.если вы понимаете русский язык просьба написать Nix56yffff127.

**Reply by boxgaming** [reply]
I'm afraid I don't speak Russian.  What browser and version are you using?

**Reply by GT9866** [reply]
yandex RUS

**Reply by boxgaming** [reply]
Are you on the latest update of the browser? Do you have the same issue if you try it in a different browser?

**Reply by GT9866** [reply]
yes

**Reply by GT9866** [reply]
try to run the file, I will send it to you by Email.

**Reply by boxgaming** [reply]
You can attach it here or open a new issue.

**Reply by GT9866** [reply]
sorry, but the file is licensed.
I can only send it to vk,EMail.
VK:  https://vk.com/art_com_ru_org
E mail: error78art@yandex.ru

**Reply by boxgaming** [reply]
Sorry @GT9866, this is open source software, we do not have private email support. Your best bet is to share the problematic portion of your program here or in a new issue so the community can help you.

**Reply by GT9866** [reply]
boxgaming can you please add 'RUN' to QBJS

**Comment by GT9866** [comment]
file.bas not supported for github

**Reply by boxgaming** [reply]
You could try attaching it as a *.txt file.

**Reply by GT9866** [reply]
ok

**Reply by GT9866** [reply]
deleting a file after 15 minutes.

**Reply by boxgaming** [reply]
It ran for me.  Looks like it might be an issue with the Sound command.  There was a fix in the latest version (#119).  You may need to make the duration shorter for each sound that you play.  In order to be more compatible with QBasic, the Sound method also waits until the sound has finished playing.  So, you will probably not need the following calls to Sleep anymore.

**Reply by GT9866** [reply]
and before that, everything was working, but he started loading further than the screen saver?

**Reply by GT9866** [reply]
![2025-01-05_20-48-45 (1)](https://github.com/user-attachments/assets/4720a12e-8cb2-471c-9f77-ae75c7d2fa01)
Did he load up to this point?

**Reply by boxgaming** [reply]
Yes

**Reply by GT9866** [reply]
So there's something wrong with my device.

**Comment by GT9866** [comment]
file deleted
